src/components/UploadPopUp/UploadPopUp.module.css

```
@@ -40,3 +40,7 @@
   justify-content: center;
   -webkit-justify-content: center;
 }
+
+.upload-Error {
+  margin:auto;
```

Just a very tiny minor formatting enhancements, it would better if there was a space right after the colon to match the overall style of the file. 

```suggestion
  margin: auto;
```